xdsl/dialects/emitc.py

```diff
@@ -87,6 +97,24 @@ def print_parameters(self, printer: Printer) -> None:
             printer.print_attribute(self.element_type)
 
 
+def is_supported_integer_type(type_attr: Attribute) -> bool:
+    """Check if an IntegerType is supported by EmitC."""
+    if isinstance(type_attr, IntegerType):
+        return type_attr.width.data in [1, 8, 16, 32, 64]
+    return False
+
+
+def is_supported_emitc_type(type_attr: Attribute) -> bool:
+    """Check if a type is supported by EmitC."""
+    if isinstance(type_attr, IntegerType):
+        # is_supported_integer_type handles the width check
+        if is_supported_integer_type(type_attr):
+            return True
+        return False
+
+    return True
```

Comment:
this is temporary and needs to return False once the all the types used in the tests are added to the function